### DISCUSSION TITLE
Error with naming variable and save it inside database inside curly braces

### DISCUSSION THREAD

**Discussion by Abdelhammied** [laravel/framework#33217] [General]
- Laravel Version: 7.10.3
- PHP Version: 7.4.7
- Database Driver & Version: MySQL 14.14 Distrib 5.7.30, for Linux

### Description:
I am passing data into my view with a key of  ``` $item ``` then when creating data into the same model with input value ``` {{ $item }} ``` an error at the console appeared ``` {{ $item }} not defined ``` 
and my view failed with out throw error at the ignition ^_^
![Screenshot from 2020-06-15 15-14-16](https://user-images.githubusercontent.com/35538743/84661698-0456da00-af1b-11ea-8fe9-c005ea308e00.png)
![Screenshot from 2020-06-15 15-13-59](https://user-images.githubusercontent.com/35538743/84661700-05880700-af1b-11ea-8d2e-861cf7d7e4fe.png)
![Screenshot from 2020-06-15 15-13-20](https://user-images.githubusercontent.com/35538743/84661705-06209d80-af1b-11ea-897e-4af7a93b36de.png)

**Comment by driesvints** [comment]
Seems like a support question so moving to discussions.
